std/json.d

```diff
@@ -167,17 +167,17 @@ struct JSONValue
     /// Value getter/setter for $(D JSON_TYPE.STRING).
     /// Throws: $(D JSONException) for read access if $(D type) is not
     /// $(D JSON_TYPE.STRING).
-    @property inout(string) str() inout pure
+    @property inout(string) str() inout pure @trusted
```

Comment:
This is an issue from the old code, but `inout` is meaningless here. The following prototype is just as good:
```d
@property string str() const pure @trusted;
```